client/app/bundles/comments/rescript/CommentForm/forms/HorizontalForm.res

```diff
@@ -31,7 +43,8 @@ let make = (~author, ~handleAuthorChange, ~text, ~handleTextChange, ~handleSubmi
         type_="submit"
         className="self-start px-3 py-1 font-semibold border-0 rounded text-sky-50 bg-sky-600 hover:bg-sky-800"
         value="Post"
+        ariaLabel="Submit comment"
       />
     </div>
   </form>
-}
+}
```

Comment:
Your editor seems to be removing the empty line at the end of each file, but only in this PR.

Please make sure each of the files end with an empty line, which will remove the red circle symbol above.